docs/install-upgrade.md

```diff
@@ -20,6 +21,58 @@ Before you install the MongoDB Community Kubernetes Operator, you must:
    ```
    git clone https://github.com/mongodb/mongodb-kubernetes-operator.git
    ```
+4. Review the possible Operator [deployment scopes](#understand-mongodb-community-operator-deployment-scopes) and configure the Operator to watch other namespaces, if necessary.
+
+### Understand Deployment Scopes
+
+You can deploy the MongoDB Community Kubernetes Operator with different scopes based on where you want to deploy Ops Manager and MongoDB Kubernetes resources:
+
+- [Operator in Same Namespace as Resources](#operator-in-same-namespace-as-resources)
+- [Operator in Different Namespace Than Resources](#operator-in-different-namespace-than-resources)
+- [Cluster-Wide Scope](#cluster-wide-scope)
+
+#### Operator in Same Namespace as Resources
+
+You scope the Operator to a namespace. The Operator watches MongoDB resources in that same [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+This is the default scope when you install the Operator using the [installation instructions](#procedure).
+
+#### Operator in Different Namespace Than Resources
+
+You scope the Operator to a namespace. The Operator watches MongoDB resources in other namespaces.
+
+To configure the Operator to watch resources in other namespaces:
+
+1. In the Operator [resource definition](../deploy/operator/operator.yaml), set the `WATCH_NAMESPACE` environment variable to one of the following values:
+
+   - the namespace that you want the Operator to watch, or
+   - `*` to configure the Operator to watch all namespaces in the cluster.
+
+   ```yaml
+       spec:
+         containers:
+           - name: mongodb-kubernetes-operator
+             image: quay.io/mongodb/mongodb-kubernetes-operator:0.5.0
+             command:
+               - mongodb-kubernetes-operator
+             imagePullPolicy: Always
+             env:
+               - name: WATCH_NAMESPACE
```

Comment:
Can you highlight lines in markup? Might be helpful here if so, but definitely not necessary.

Reply:
I don't think there's a simple way to do this. I did a little research but came up empty.